operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -598,11 +664,21 @@ function createPrimordialDomain() {
 
     diff_rc=$(cat /tmp/model_diff_rc)
     rm ${DECRYPTED_MERGED_MODEL}
-    trace "createPrimordialDomain: model diff returns "${diff_rc}
-
+    trace "createPrimordialDomain: model diff return code list (can be empty): "${diff_rc}
 
     local security_info_updated="false"
+    local cannot_perform_online_update="false"
     security_info_updated=$(contain_returncode ${diff_rc} ${SECURITY_INFO_UPDATED})
+    cannot_perform_online_update=$(contain_returncode ${diff_rc} ${NOT_FOR_ONLINE_UPDATE})
+
+    if [ ${cannot_perform_online_update} == "true" ] ; then
+      trace SEVERE "Model in image online update failed because of forbidden changes:  Domain resource specified " \
```

Comment:
Model in image --> Model in Image, Domain resource -> The domain resource, etc.  Lets meet to word smith this a bit...

Reply:
Let me know if it is OK to change this particular SEVERE to:
```
+      trace SEVERE \
+      "Model in Image online update failed because of forbidden changes:" \
+      " the Domain resource specified 'spec.configuration.model.onlineUpdate.enabled=true'," \
+      " but the model changes cannot use online update - such as:" \
+      " changing ListenPort, ListenAddress, SSL, changing top level Topology attributes," \
+      " or deleting a ServerTemplate."
```

Reply:
Changed.